test/routes/setup_test.js

```diff
@@ -0,0 +1,18 @@
+let path = require('path');
```

Comment:
I tried, but the setup.js file doesn't seem to get a mocha test context, so the globals to `setup` and `teardown` don't exist.